lib/bootsy/activerecord/image_gallery.rb

```diff
@@ -10,8 +10,13 @@ module Bootsy
   # that do not point to resources older than the given time
   # limit.
   class ImageGallery < ActiveRecord::Base
-    belongs_to :bootsy_resource, polymorphic: true, autosave: false,
-                                 optional: true
+    if Rails::VERSION::STRING.split(".").first.to_i >= 5
```

Comment:
The latest release still supports Rails 4. Apps on Rails 4 can lock on that version. But unless there is a security fix we're moving forward with Rails 5 only.